worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/PartitionDataWriter.java

```diff
@@ -549,8 +549,10 @@ protected synchronized long close(
       }
     }
     if (diskFileInfo != null) {
+      source.updateHistogram(WorkerSource.PARTITION_FILE_SIZE(), diskFileInfo.getFileLength());
       return diskFileInfo.getFileLength();
     } else {
+      source.updateHistogram(WorkerSource.PARTITION_FILE_SIZE(), memoryFileInfo.getFileLength());
```

Comment:
The metrics were removed by https://github.com/apache/celeborn/commit/951b626a9896759a9ce09add39358ffd08d78496#diff-93aed69b393af59cefdfa6f5293f4dfb9cba96a9be23f3eec0bbe7d61f6d65be
<img width="1036" height="141" alt="image" src="https://github.com/user-attachments/assets/c94f731c-79af-4421-8d19-3f3e27f86fb0" />